packages/fxa-auth-server/lib/senders/emails/templates/postVerify/index.mjml

```diff
@@ -17,14 +17,17 @@
 <mj-section>
   <mj-column>
     <mj-text css-class="text-body">
-      <span data-l10n-id="postVerify-description">Sync privately keeps your bookmarks, passwords and other Firefox data the same across all your devices.</span>
+      <span data-l10n-id="postVerify-description-2">It’s easy! Just install Firefox on another device and log in to sync. It’s like magic!</span>
+    </mj-text>
+    <mj-text css-class="text-body">
+      <span data-l10n-id="postVerify-sub-description">(Psst...It also means you can get your bookmarks, passwords, and other Firefox data everywhere you’re signed in.)</span>
```

Comment:
Just adding a note confirming that a smart ellipsis is used in this .txt file since it doesn't look like it. :)